redaxo_loader.php

```diff
@@ -1,9 +1,53 @@
 <?php
+error_reporting(E_ALL);
+ini_set("display_errors", 1);
+
+
 /**
- * Download latest REDAXO release from github
+ * Download REDAXO releases from github
  * License: MIT License
  */
 
+
+// set loader name
+$loader_name = 'redaxo_loader.php';
+
+// check requirements
+$required = [];
+
+if (basename(__FILE__) != $loader_name) {
+    $required[] = 'Der Dateiname des Loaders ist nicht'.$loader_name;
+}
+
+if (!in_array('curl', get_loaded_extensions())) {
+    $required[] = 'Die Class curl wurde nicht gefunden';
+}
+
+if (!in_array('zip', get_loaded_extensions())) {
+    $required[] = 'Die Class zip wurde nicht gefunden';
+}
+
+if (!function_exists('json_decode')) {
+    $required[] = 'Die Funktion json_decode wurde nicht gefunden';
+}
+if (!function_exists('file_put_contents')) {
+    $required[] = 'Die Funktion file_put_contents wurde nicht gefunden';
+}
+
+if (count($required) > 0) {
+    echo '<h2>Fehler: Voraussetzungen nicht erfüllt</h2>';
+    echo '<ul><li>' . implode('</li><li>', $required) . '</li></ul>';
+    exit();
+}
+
+function checkUrl($url) {
```

Comment:
Gibt es einen grund warum du das nicht in curl_get_file_content machen willst?

Reply:
Wir wollten es so früh wie möglich abfragen und so jeden Output unterdrücken. Ich schaue mal ob‘s auch da passieren kann

Reply:
da hier 2 unterschiedliche URLs an die Funktion übergeben werden mit unterschiedlicher Schreibweise, denke ich das @danspringer Lösung schon passt. Sie greift frühzeitig und es kommt nicht zu einem Output des HTMLs